Userland/Libraries/LibCore/System.cpp

```diff
@@ -403,7 +405,7 @@ ErrorOr<int> anon_create([[maybe_unused]] size_t size, [[maybe_unused]] int opti
         TRY(close(fd));
         return Error::from_errno(saved_errno);
     }
-#elif defined(AK_OS_MACOS) || defined(AK_OS_EMSCRIPTEN) || defined(AK_OS_OPENBSD) || defined(AK_OS_NETBSD)
+#elif defined(AK_OS_MACOS) || defined(AK_OS_EMSCRIPTEN) || defined(AK_OS_OPENBSD) || defined(AK_OS_NETBSD) || defined(AK_OS_SOLARIS)
```

Comment:
generic BSD support?

Reply:
FreeBSD has another way of handling this,but as that's defined further up,that one could be really changed to BSD_GENERIC to catch the rest after having checked for FreeBSD already.
I'll change that tomorrow,only using the laptop right now.